src/commands/schemas.test.ts

```diff
@@ -23,11 +23,32 @@ import { SchemaRegistry } from "../models/schemaRegistry";
 import { KafkaTopic } from "../models/topic";
 import {
   CannotLoadSchemasError,
+  copySchemaRegistrySubject,
   determineLatestSchema,
   diffLatestSchemasCommand,
   getLatestSchemasForTopic,
 } from "./schemas";
 
+describe("copySchemaRegistrySubject", () => {
+  let _originalClipboardContents: string | undefined;
+
+  beforeEach(async () => {
+    _originalClipboardContents = await require("vscode").env.clipboard.readText();
```

Comment:
We should be able to import `env` from `vscode` directly (like `commands`), so we shouldn't need any of these `require()` imports